packages/util-dev/package.json

```diff
@@ -53,6 +53,7 @@
   "devDependencies": {
     "@types/jest": "^26.0.24",
     "eslint": "^7.32.0",
+    "get-port-please": "^2.5.0",
```

Comment:
This should be in `dependencies` since it's used in the library code (`src/`). No need for a fixup for this one.